internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -352,7 +352,14 @@ func (u *ALMExamplesUpdate) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) err
 		}
 	}
 	sb.WriteString(`]`)
-
-	csv.GetAnnotations()["alm-examples"] = sb.String()
+	format, _ := prettyPrint([]byte(sb.String()))
+	csv.GetAnnotations()["alm-examples"] = string(format)
 	return nil
 }
+
+// prettyPrint returns a print json
+func prettyPrint(b []byte) ([]byte, error) {
+	var out bytes.Buffer
+	err := json.Indent(&out, b, "", "  ")
+	return out.Bytes(), err
```

Comment:
I think we could return the string directly:
```suggestion
	return out.String(), err
```

Reply:
Done tks for the review